Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -400,6 +400,11 @@ NonnullRefPtr<Statement> Parser::parse_statement(AllowLabelledFunction allow_lab
     case TokenType::Semicolon:
         consume();
         return create_ast_node<EmptyStatement>({ m_state.current_token.filename(), rule_start.position(), position() });
+    case TokenType::Slash:
+    case TokenType::SlashEquals:
+        // Lexer was wrong this is a regexexpr
```

Comment:
Please write comments more carefully, as whole sentences including punctuation. See https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#comments.

Also, `regexexpr` is weird - the `ex` in `regex` already stands for expression...